.idea/django.iml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Oops, you probably want to add `.idea` to `.git/info/exclude` or `core.excludesFile` per https://git-scm.com/docs/gitignore.